helm/gke-example/values.yaml

```diff
@@ -29,11 +29,12 @@ secret:
     encryptionKey:
 
 
-#apiSettings:
+# apiSettings:
+#   dnsName: your-api.hostname.here
```

Comment:
this isn't actually used in the GKE example, is it?

Reply:
Oops. Yeah... I removed that.